YuliiaShaposhnikova/HW9/Task1.py

```diff
@@ -0,0 +1,28 @@
+from random import randint
```

Comment:
good job
look at the more optimized version of the code

import random

def guess_my_number():
    """
    Function guesses number from 1 to 100 and asks user guess this number with 10 attempts.
    """
    guess_number = random.randint(1, 100)
    count_tries = 1

    user_number = int(input("I guessed a number from 1 to 100. Guess my number: "))

    for count_tries in range(1, 11):
        if guess_number < user_number:
            user_number = int(input("No, my number is less. Try again: "))
        elif guess_number > user_number:
            user_number = int(input("No, my number is more. Try again: "))
        elif guess_number == user_number:
            print("Congratulations! You win!")
            break

    if count_tries == 10 and guess_number != user_number:
        print("Oh no! You have run out of attempts")